narayana-release-process.sh

```diff
@@ -228,25 +221,6 @@ then
   exit
 fi
 
-if [[ $QUAYIOOK != n* ]]
-then
-  echo "building, tagging and pushing podman images to quay.io ..."
-#read -p "Have you logged in (podman login quay.io), if not you will be prompted (y/n): " ok
-#if [[ $ok == y* ]]; then
-  # build the lra-coordinator image
-  podman build --tag lra-coordinator --build-arg NARAYANA_VERSION=${CURRENT} .
-  # tag it and push it
-  podman tag lra-coordinator quay.io/jbosstm/lra-coordinator:${CURRENT}
-  podman tag lra-coordinator quay.io/jbosstm/lra-coordinator:latest
-
-  # push the image
-  podman push quay.io/jbosstm/lra-coordinator:${CURRENT}
-  podman push quay.io/jbosstm/lra-coordinator:latest
-  # the image will appear in https://quay.io/repository/jbosstm/lra-coordinator?tab=tags
-#else
-#  echo "alternatively run the above commands manually"
-#fi
-fi
 
 # not sure why we need to look at CI at this point so commenting it out
-# xdg-open http://narayanaci1.eng.hst.ams2.redhat.com/ &
+# xdg-open https://ci-jenkins-csb-narayana.apps.ocp-c1.prod.psi.redhat.com/ &
```

Comment:
I think delete these lines but you could clarify why we want to go to CI (which is to run-post release quickstarts job to verify things are working and to obtain performance numbers for the release and that can be printed out to help the releaser to perform those steps.

So basically remove these lines and change "# not sure why we need to look at CI at this point so commenting it out" -> "echo "Please visit Narayana CI and check the quickstarts are working with the release and obtain performance numbers for the blog post"